VocabularyWord.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  File.swift
+//  Swift Vocabulary
+//
+//  Created by Clean Mac on 4/8/20.
+//  Copyright © 2020 LambdaStudent. All rights reserved.
+//
+
+import Foundation
+
+struct VocabularyWord {
+    word: String
+    definition: String 
```

Comment:
Remember we have to assign a variable with `var` in front. example `var word: String`